src/objects.js

```diff
@@ -1,41 +1,50 @@
 const createPerson = (name, age) => {
-  // your code here
+return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object.age;
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
-};
+  return person.age > 65
+  };
 
 const getAges = people => {
-  // your code here
+  return people.map(people => people.age);
```

Comment:
```suggestion
  return people.map(person => person.age);
```
As detailed earlier, naming conventions make the code easier to understand